configuration/virtual-fleet-management/scenarios/default/virtual_vehicle_3/scenario1.json

```diff
@@ -0,0 +1,158 @@
+{
+  "map": "map.osm",
+  "starting_station": "Těšínská",
+  "missions": [
+    {
+      "delay_seconds": 0,
+      "name": "mission1",
+      "stops": [
+        {
+          "name": "Svatopluka Čecha A"
+        }
+      ],
+      "route": "Moravské náměstí 2"
+    },
+    {
+      "delay_seconds": 0,
```

Comment:
Put some delay, otherwise the mission1 is cancelled immediately and the car never moves